packages/shared-ui/src/components/header/ExternalLink.tsx

```diff
@@ -0,0 +1,10 @@
+export function ExternalLink({
```

Comment:
This component needs to be moved to common UI from header once it's moved to shared-ui